docs/ROADMAP.md

```diff
@@ -47,11 +47,49 @@ concepts.
 ### 6. Function catalog
 
 All of the hydration and validation logic has been moved from the kpt binary 
-to functions allowing for flexibility and security.  This enables new 
+to functions allowing for flexibility and security. This enables new 
 scenarios like limiting the customization and validation to a subset of 
 allowed functions.  The function catalog has received additional functions, 
 examples and help. Please visit the [function catalog] for further information.
 
+## In Progress
+
+### 7. Resource filtering in `kpt fn` commands
+
+Users want to invoke a kpt function (imperatively and declaratively) on a subset of 
+resources in the package by selecting them on the basis of GVKNN, package-path, 
+file path etc. For example, set work-load identity annotation on all Kubernetes 
+Service Account resources in this package. [Tracker](https://github.com/GoogleContainerTools/kpt/issues/2015)
+
+### 8. Integrate kpt with Cloud Code
+
+One of the major areas of investment is to integrate Cloud Code with kpt to provide 
```

Comment:
Please add a link to cloud code docs. Maybe we can use https://cloud.google.com/code or https://cloud.google.com/code/docs for now.

Reply:
Forgot doing that. Thanks for pointing. Done.